zh/backup-to-gcs-br.md

```diff
@@ -0,0 +1,227 @@
+---
```

Comment:
The gcs docs for lightning+mydumper have updated, please update the docs based on the latest docs.

Reply:
done